.vib/superset/goss/goss.yaml

```diff
@@ -0,0 +1,31 @@
+# Copyright Broadcom, Inc. All Rights Reserved.
+# SPDX-License-Identifier: APACHE-2.0
+
+http:
+  http://superset-web:80/health:
```

Comment:
In other charts we also test the container port, using a different port from the default and doing http://127.0.0.1:{{vars.containerPorts.http}}